### PR TITLE
Bumped some versions and adjusted some imports to keep py3 happy

### PR DESCRIPTION
**Pull request by seanwiseman**

### CODE REVIEW

**Comment by lsh-0** [comment]
pinging @giorgiosironi , article-scheduler is still a second class citizen on elife-dashboard and we still rely on the cron job to call `./install.sh` to bump those requirements: https://github.com/elifesciences/elife-dashboard-formula/blob/master/salt/elife-dashboard/scheduler.sls#L151

this broke once in recent memory

File: install.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e # everything must succeed.
 if [ ! -d venv ]; then
-    virtualenv --python=`which python2` venv
+    virtualenv --python=`which python3.5` venv
```
Comment (giorgiosironi) [inline install.sh]: This install.sh script will use `python3.5` in a new virtualenv, but leave in place existing `python2` virtualenvs. Should either keep installing python2 by default and override it in some way if we want to support both; or check the current venv for the presence of `python3.5`.

Comment (giorgiosironi) [reply install.sh]: Also, the current dashboard stacks do not install python3: https://github.com/elifesciences/builder-private/blob/master/salt/top.sls#L25
e.g. Lax does it in this way: https://github.com/elifesciences/builder-private/blob/master/salt/top.sls#L96

**Comment by giorgiosironi** [comment]
Once merged in master, this will be applied to all `elife-dashboard--*` stacks so it's recommend to run `test-elife-dashboard` and afterwards `prod-elife-dashboard` to immediately apply the change and check if everything's fine.

File: install.sh (hunk 1 above)
Comment (lsh-0) [reply install.sh]: good catch, I overlooked that. this is how lax does it @seanwiseman : https://github.com/elifesciences/lax/blob/develop/install.sh#L7-L15

Comment (seanwiseman) [reply install.sh]: Thank you, both.

I have just replicated what you do in lax @lsh-0 . I will put the builder-private PR together now.

**Comment by giorgiosironi** [comment]
Notice this project has no build, so it relis on people running tests locally.

**Comment by seanwiseman** [comment]
Yes, considering the changes in the near future I did not think it was worth putting together an automated build at this stage (considering it was not done previously). Though yes, it now relies on responsible local testing before commits.

**Pull request merged**